blog/_posts/2017-07-31-team-impactdevs-part-one.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Team ImpactDevs Amsterdam - Part 1
+layout: post
+created_at: Tue Jul 31 2017
+permalink: blog/2017-07-31-team-impactdevs
+current: blog
+author: Jess
+twitter: teamimpactdevs
+categories:
+- student-posts
+---
+
+![image](../img/blog/2017/2017-07-teamimpactdevs.jpg)
+<div class="image-credits">Team ImpactDevs</div>
```

Comment:
could you add here an image credit for who took the picture/who it belongs to/who should be credited if we use it again? (e.g. `(Image: Team ImpactDevs)` or `(Image: Name-of-your-coach)`)